packages/edge-bundler/deno/vendor/deno.land/x/eszip@v0.55.2/eszip.ts

```diff
@@ -0,0 +1,189 @@
+// CLI utility to build/list/extract/run ESZIPs
```

Comment:
We've vendoring the `eszip` module (so we don't do any remote imports at runtime), but somehow this file was missing. It's needed for the extraction logic, so I'm adding it here.